src/styles/page.scss

```diff
@@ -440,3 +440,83 @@ div[data-tabs] input[type="radio"]:checked + label {
 div[data-tabs] input[type="radio"]:checked + label + div[data-tab] {
   display: block;
 }
+
+/* Interactive checkbox */
```

Comment:
I am wondering if these styles should live in a separate scss partial. I will leave it as it is and we can change it later if you agree 😃